src/bosh-director/lib/bosh/director/api/controllers/disks_controller.rb

```diff
@@ -20,7 +20,12 @@ class DisksController < BaseController
 
       delete '/:orphan_disk_cid' do
         job_queue = JobQueue.new
-        task = Bosh::Director::Jobs::DeleteOrphanDisks.enqueue(current_user, [params[:orphan_disk_cid]], job_queue)
+        orphan_param = params['orphan']
+        if orphan_param.nil? || orphan_param == 'false'
+          task = Bosh::Director::Jobs::DeleteOrphanDisks.enqueue(current_user, [params[:orphan_disk_cid]], job_queue)
+        else
+          task = Bosh::Director::Jobs::OrphanDiskJob.enqueue(current_user, params[:orphan_disk_cid], job_queue)
+        end
```

Comment:
Chatted about this more. Ignore my comment.

Long-term, the committed code seems like the most practical approach (though I still think it's going to be a slightly confusing API outside of the CLI).